Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/SAMLAssertionBuilder.java

```diff
@@ -44,8 +44,14 @@
     // Authorization Decision Action is always set to Execute
     static final String AUTHZ_DECISION_ACTION_EXECUTE = "Execute";
 
+    static final String AUTHN_SESSION_INDEX = "123456";
+
     // Valid Authorization Decision values
     static final String AUTHZ_DECISION_PERMIT = "Permit";
+    static final String X509_NAME_ID = "urn:oasis:names:tc:SAML:1.1:nameid-format:X509SubjectName";
+    static final String X509_AUTHN_CNTX_CLS = "urn:oasis:names:tc:SAML:2.0:ac:classes:X509";
+    static final String UNSPECIFIED_AUTHN_CNTX_CLS = "urn:oasis:names:tc:SAML:2.0:ac:classes:unspecified";
```

Comment:
check code format.  Make this as 1 line for easy to read.  Any reason why we re-arrange variable?